Contest_API.md

```diff
@@ -534,6 +529,34 @@ event feed, also note that these should not be considered proper REST
 endpoints, and that the `event-feed` endpoint is a streaming feed in
 NDJSON format.
 
+Note that `api`, `access`, `account`, `state`, `scoreboard`, and `event-feed`
+are singular nouns and indeed contain only a single object.
+
+### Required and optional endpoints
+
+The only required endpoints are metadata: `api` and `access`.
+The only requirements for properties are that collections must have
+an `id` property, and all endpoints that are supported must
+have at least some property, i.e. an endpoint can't be empty.
```

Comment:
Is it clear (enough) that this property could be the `id`? I.e. for collections the "and all endpoints that are supported must have at least some property" requirement is actually a no-op?

Reply:
How would you word it? I'm not sure there will be cases where only an id will be useful, so I kind of felt like it was ok if this was technically valid but not obvious that was the case :). Or that we should even enforce one property in addition to the id.

Reply:
If anything I would lean the other way, i.e. drop ", and all endpoints that are supported must
have at least some property, i.e. an endpoint can't be empty." 

Having an empty endpoint is useless, but it doesn't break anything. (Also, there is a trivial, and similarly useless, workaround. You could add the "ignoreme" property to an otherwise empty endpoint.).

Reply:
Ok, fair enough. Suggested changes committed.